frontend/src/styles/vars.scss

```diff
@@ -118,6 +118,7 @@ $_lifecycle_dormant: #f86234;
     // Special case color for primary buttons with a nested primary button
     --primary-button-nested-hover: #d2dbff;
 
+    --bg-side: var(--side);
```

Comment:
Interesting... Is this not covered by the utility builder (it should add `--bg-side` and `--text-side` since I moved it there...)

Reply:
`--bg-side` was undefined so I guess not :)

👀 now

Reply:
Ah, this? https://github.com/PostHog/posthog/blob/master/frontend/src/styles/utilities.scss#L228

Creates a class not a var. Do you know which it should be?